src/community/schemaless-features/mongodb-schemaless/src/test/java/org/geoserver/schemalessfeatures/mongodb/FeaturesMongoTest.java

```diff
@@ -0,0 +1,80 @@
+package org.geoserver.schemalessfeatures.mongodb;
```

Comment:
Lacks copyright header but... wondering if it actually belongs in this PR?

Reply:
removed